modules/localization/msf/local_integ/measure_republish_process.h

```diff
@@ -0,0 +1,103 @@
+/******************************************************************************
+ * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file measure_republish_process.h
+ * @brief The class of MeasureRepublishProcess
+ */
+
+#ifndef MODULES_LOCALIZATION_MSF_MEASURE_REPUBLISH_PROCESS_H_
+#define MODULES_LOCALIZATION_MSF_MEASURE_REPUBLISH_PROCESS_H_
+
+#include <pthread.h>
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+#include <list>
+#include <string>
+#include "modules/common/status/status.h"
+#include "modules/drivers/gnss/proto/gnss_best_pose.pb.h"
+#include "modules/localization/proto/localization.pb.h"
+#include "modules/localization/msf/common/util/frame_transform.h"
+#include "modules/localization/msf/local_integ/localization_params.h"
+#include "include/sins_struct.h"
+
+// #include "local_sins/frame_transform.hpp"
+// #include "local_sins/integrated_navigation.hpp"
+
+/**
+ * @namespace apollo::localization
+ * @brief apollo::localization
+ */
+namespace apollo {
+namespace localization {
+namespace msf {
+
+typedef Eigen::Affine3d TransformD;
```

Comment:
Don't put an alias in your public API just to save typing in the implementation; do so only if you intend it to be used by your clients.